docs/astro/src/content/docs/reference/window/contextmenu.mdx

```diff
@@ -0,0 +1,62 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Window
+description: Window element api.
+---
+import SlintProperty from '/src/components/SlintProperty.astro';
+import Link from '/src/components/Link.astro';
+
+`ContextMenu` is an invisible element that is used to show a context menu.
+
+The context menu is shown if the user right-clicks on the area covered by the `ContextMenu` element,
+or if the user presses the "Menu" key on their keyboard while a `FocusScope` within the `ContextMenu` is focused.
+It is also possible to show the context menu by calling the `show` function on the `ContextMenu` element.
+
+In addition to normal sub elements, the `ContextMenu` element can contain `MenuItem` elements.
+The MenuItem elements are defining the actual menu.
```

Comment:
I think it's better to get straight to the point than to start about an unusual case (somebody placing an `Image` as a child).

```suggestion
Define the structure of the menu by placing `MenuItem` elements into the `ContextMenu`.
```